pkg/tests/accounts.go

```diff
@@ -0,0 +1,59 @@
+package tests
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/nimrodshn/cs-load-test/pkg/helpers"
+	"github.com/nimrodshn/cs-load-test/pkg/result"
+	vegeta "github.com/tsenart/vegeta/v12/lib"
+)
+
+func TestRegisterNewCluster(attacker *vegeta.Attacker,
+	testID string,
+	metrics map[string]*vegeta.Metrics,
+	rate vegeta.Pacer,
+	outputDirectory string,
+	duration time.Duration) error {
+
+	testName := "new-cluster-registration"
+	fileName := fmt.Sprintf("%s_%s", testID, testName)
+
+	log.Printf("Executing Test: %s", testName)
+
+	// TODO: Generate a UUID for each Request
+	// TODO: The authorization_token should be real. Not sure what to set it as, though.
+	target := vegeta.Target{
+		Method: http.MethodPost,
+		URL:    helpers.ClusterRegistrationEndpoint,
+		Body:   []byte("{\"authorization_token\": \"specify-me\", \"cluster_id\": \"c98550e5-1c9f-47bb-b46f-b2b6e7befeb3\"}"),
```

Comment:
@kurtismullins you can use the ocm sdk for this:
```
import (
        ...
 	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1" 
)
..
body, err := amsv1.NewClusterRegistrationRequest().AuthorizationToken("foo-bar").ClusterID("lebron-james").Build()
if err != nil {
   return err
}
var raw bytes.Buffer
err = v1.MarshalClusterRegistrationRequest(body, &raw)
if err != nil {
	return err
}

...
Body: raw.Bytes()
```

Reply:
Oooh nice, thank you!